docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -460,6 +460,16 @@ The integer value indicates the sampling decision that has been made for this tr
 A value of 0 indicates that the trace should not be collected, and a value of 1
 requests that spans are sampled and reported.
 
+.. _config_http_conn_man_headers_sw8:
+
+sw8
+----------
+
+The *sw8* HTTP header is used by the SkyWalking tracer in Envoy. It contains the key
+tracing context for SkyWalking tracer and is used to establish the relationship between
+tracing spans of downstream and Envoy. See more on SkyWalking tracing
+`here <https://github.com/apache/skywalking/blob/v8.1.0/docs/en/protocols/Skywalking-Cross-Process-Propagation-Headers-Protocol-v3.md>`__.
```

Comment:
Just missing a couple of 'the' articles:

for *the* SkyWalking tracer
between *the* tracing spans